lib/modules/manager/velaci/extract.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { loadFixture } from '../../../../test/util';
+import { extractPackageFile } from './extract';
+
+const invalidYAML = loadFixture('invalid.yml');
```

Comment:
please move inline when not shared by multiple tests.

also use `Fixtures` class.

Reply:
should be fixed